### DISCUSSION TITLE
Reset usage after meter swap

### DISCUSSION THREAD

**Discussion by markisoke** [dsmrreader/dsmr-reader#1472] [Questions & answers / How-to's] [answered]
> 🇳🇱 Lees eerst [deze info](https://github.com/dsmrreader/dsmr-reader/blob/v4/CONTRIBUTING.md) om jezelf te helpen.
> 
> 🇬🇧 First read [this info](https://github.com/dsmrreader/dsmr-reader/blob/v4/CONTRIBUTING.md) for self-service.

Vandaag is mijn slimme meter omgewisseld wegens de verzwaring naar 3 fase. Hierdoor zijn de standen weer bij 0 begonnen en nu vroeg ik me af om hoe ik dit kan resetten zodat ik geen negatieve waardes meer heb.

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je vraag, je kunt hier meer informatie vinden: https://dsmr-reader.readthedocs.io/nl/v4/how-to/troubleshooting/smart-meter-replacement.html

**Comment by markisoke** [comment]
> 
> 
> Bedankt voor je vraag, je kunt hier meer informatie vinden: https://dsmr-reader.readthedocs.io/nl/v4/how-to/troubleshooting/smart-meter-replacement.html

Bedankt Dennis, zojuist alles aangepast en ziet er nu weer goed uit.

**Comment by dennissiemensma** [comment]
Top!

**Comment by wtreur** [comment]
Ter info: Sinds versie 5 is de info hier te vinden: https://dsmr-reader.readthedocs.io/en/v5/how-to/troubleshooting/smart-meter-replacement.html
